samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -248,44 +248,71 @@ static struct bt_bap_stream_ops stream_ops = {
 	.recv = stream_recv_cb,
 };
 
-static void base_recv_cb(struct bt_bap_broadcast_sink *sink, const struct bt_bap_base *base)
+#if defined(CONFIG_LIBLC3)
```

Comment:
I wonder why this depends on CONFIG_LIBLC3 as the types defined in the spec are not LC3 only values.

Reply:
Because it looks for an LC3 codec configuration in the BASE. That is only something we care about if we want to decode LC3 data.